tst.c

```diff
@@ -167,19 +167,34 @@ static void *tst_del_word(tst_node **root,
     return victim; /* return NULL on successful free, *node otherwise */
 }
 
-/** tst_ins_del() ins/del copy or reference of 's' from ternary search tree.
- *  insert all nodes required for 's' in tree at eqkid node of leaf. if 'del'
- *  is non-zero deletes 's' from tree, otherwise insert 's' at node->eqkid
- *  with node->key set to the nul-character after final node in search path. if
- *  'cpy' is non-zero allocate storage for 's', otherwise save pointer to 's'.
- *  if 's' already exists in tree, increment node->refcnt. (to be used for del).
- *  returns address of 's' in tree on successful insert (or on delete if refcnt
- *  non-zero), NULL on allocation failure on insert, or on successful removal
- *  of 's' from tree.
+/** next_node() based on char '**s' find out the next node for searching given
```

Comment:
Fix the grammar errors!

Reply:
e.g. "equals to"

Reply:
Within the comment, you should mention `a pointer to a pointer to char` rather than `char '**s'`.